api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -226,7 +226,6 @@ public void deleteWorkspace(DbWorkspace dbWorkspace) {
         dbWorkspace.getWorkspaceNamespace(), dbWorkspace.getFirecloudName());
     dbWorkspace.setWorkspaceActiveStatusEnum(WorkspaceActiveStatus.DELETED);
     dbWorkspace = workspaceDao.saveWithLastModified(dbWorkspace);
-    userRecentWorkspaceDao.deleteByWorkspaceId(dbWorkspace.getWorkspaceId());
```

Comment:
Could you also delete the `deleteByWorkspaceId` function in `userRecentWorkspaceDao` that is now unused?

Reply:
right. good catch